src/components/proposal/actions/WhitelistAction.stories.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * This program has been developed by students from the bachelor Computer Science at Utrecht University within the Software Project course.
+ * © Copyright Utrecht University (Department of Information and Computing Sciences)
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Accordion } from '@/src/components/ui/Accordion';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { WhitelistAction } from './WhitelistAction';
+
+const meta = {
+  component: WhitelistAction,
+  argTypes: {
+    action: {
+      table: {
+        disable: true,
+      },
+    },
+    asChild: {
+      table: {
+        disable: true,
+      },
+    },
+    value: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+} satisfies Meta<typeof WhitelistAction>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    value: 'first',
```

Comment:
Can you extract these values to a variable `dummyWhitelistAction` and add that to the `useProposal.ts` file (where the other dummy actions are also located)?

Reply:
While you're at it, you can also then add the dummy action to the `ProposalActions.stories.tsx`